### PR TITLE
Remove ids and grammar changes.

### PR DESCRIPTION
**Pull request by capslock321**
## Summary
# IDS
Specific discord ids was in the example files. Those who just want to try the program will be faced with discord.Forbidden if they forget to change the ids. This fixes this minor inconvenience by removing the ids. 
# Grammar
There are specific comments that are lacking periods or are just grammatically incorrect. Although comments do not have to have capitalization and periods, it is best to add them to bring forth a sense of professionalism.

Also functions in a class are called methods.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

File: examples/views/button_roles.py [hunk 1]
```diff
@@ -26,80 +26,78 @@ def __init__(self, role: discord.Role):
...
     @commands.Cog.listener()
     async def on_ready(self):
-        """This function is called every time the bot restarts.
+        """This method is called every time the bot restarts.
         If a view was already created before (with the same custom IDs for buttons)
         it will be loaded and the bot will start watching for button clicks again.
         """
-
-        # we recreate the view as we did in the /post command
+
```
Comment (Lulalaby) [inline examples/views/button_roles.py:89]:
```suggestion

```

**Review by Lulalaby — APPROVED** [review]

File: examples/views/button_roles.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 # this is the list of role IDs that will be added as buttons.
 role_ids = [
-    846383888003760217, 846383888036134932
+
```
Comment (Dorukyum) [inline examples/views/button_roles.py:16]:
```suggestion
    ...
```

File: examples/views/button_roles.py [hunk 3]
```diff
@@ -26,80 +26,77 @@ def __init__(self, role: discord.Role):
...
+    and to initialize the view again when the bot is restarted.
     """
 
     def __init__(self, bot):
         self.bot = bot
 
-    # make sure to set the guild ID here to whatever server you want the buttons in
-    @slash_command(guild_ids=[846383887973482516], description="Post the button role message")
+    # Make sure to provide a list of guild ids in the guild_ids kwarg argument.
+    @slash_command(guild_ids=[], description="Post the button role message")
```
Comment (Dorukyum) [inline examples/views/button_roles.py:67]:
```suggestion
    @slash_command(guild_ids=[...], description="Post the button role message")
```

File: examples/views/button_roles.py [hunk 4]
```diff
@@ -26,80 +26,77 @@ def __init__(self, role: discord.Role):
...
         it will be loaded and the bot will start watching for button clicks again.
         """
-
-        # we recreate the view as we did in the /post command
+        # We recreate the view as we did in the /post command.
         view = discord.ui.View(timeout=None)
-        # make sure to set the guild ID here to whatever server you want the buttons in
-        guild = self.bot.get_guild(846383887973482516)
+        # Make sure to set the guild ID here to whatever server you want the buttons in!
+        guild = self.bot.get_guild()
```
Comment (Dorukyum) [inline examples/views/button_roles.py:92]:
```suggestion
        guild = self.bot.get_guild(...)
```

**Review by Dorukyum — COMMENTED** [review]
I think it's better to use `...` to show that people should add values.

**Review by Lulalaby — APPROVED** [review]

File: examples/views/button_roles.py [hunk 5]
```diff
@@ -24,80 +24,77 @@ def __init__(self, role: discord.Role):
...
 
     async def callback(self, interaction: discord.Interaction):
-        """This function will be called any time a user clicks on this button
+        """This function will be called any time a user clicks on this button.
 
         Parameters
         ----------
         interaction : discord.Interaction
-            The interaction object that was created when the user clicked on the button
+            The interaction object that was created when the user clicked on the button.
```
Comment (RPSMain) [inline examples/views/button_roles.py:32]:
```suggestion
            The interaction object that was created when a user clicks on a button.
```

**Review by RPSMain — CHANGES_REQUESTED** [review]

**Review by RPSMain — APPROVED** [review]

**Pull request merged**